### DISCUSSION TITLE
Anti-alias option has been removed - why?

### DISCUSSION THREAD

**Discussion by MikeRich88** [HexFiend/HexFiend#456] [General]
2.17 release notes - "Anti-alias option has been removed"

How come? I would use this option in certain circumstances. (Non-Retina screens for example)

I understand if it has to do with the OS taking it away/making it extremely difficult or something along those lines.

**Comment by MikeRich88** [comment]
https://developer.apple.com/documentation/coregraphics/1455178-cgcontextsetshouldantialias

CGContextSetShouldAntialias does not appear to be deprecated or anything.

https://github.com/HexFiend/HexFiend/commit/30ae944c6237268abe7d7ae57c30b1a7d4d355ee

**Comment by kainjow** [comment]
Apple has been shipping Retina displays for over a decade now. It's going to a very small niche group that wants anti-aliasing for non-Retina displays now.

**Reply by MikeRich88** [reply]
The 2015 non-Retina model was shipped until as late as 2019, which is only ~4 years ago. This severely compromises your argument.

**Reply by MikeRich88** [reply]
Not to mention the fact that even with the latest most brand new model (such as Mac mini) anyone can at any time connect a generic non-Retina display and may appreciate such capability for.
